src/core/dbus-execute.c

```diff
@@ -1825,6 +1826,9 @@ int bus_exec_context_set_transient_property(
         if (streq(name, "LogRateLimitBurst"))
                 return bus_set_transient_unsigned(u, name, &c->log_ratelimit_burst, message, flags, error);
 
+        if (streq(name, "LogFilterRegex"))
+                return bus_set_transient_string(u, name, &c->log_filter_regex, message, flags, error);
```

Comment:
Is it not necessary to set xattr here when the property is set??

Reply:
My understanding is that `bus_set_transient_string()` in this situation will be call to set `LogFilterRegex` to an `ExecContext`, same as when systemd reads a unit file and fill the `ExecContext`. Setting the xattr is performed at a later time when the cgroup is created/updated. As systemd already create cgroups for transient unit, I expect it to also set the xattr properly.

[As you can see](https://github.com/systemd/systemd/pull/24058#discussion_r931466470), xattr are set as expected for transient units.